src/containers/card.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../css/card.css';
+
+
+function Cards () {
+    return (
+        <div>
+            <div className="row tile_count">
+				<div className="col-md-2 col-sm-4 col-xs-6 tile_stats_count">
```

Comment:
Move this to component folder
should contain template for single card component